LINTING.md

```diff
@@ -0,0 +1,81 @@
+# Linting Standard
+
+This standard outlines the minimum [linting](https://en.wikipedia.org/wiki/Lint_(software)) expecations for Dissemination repos.
+
+## Context
+
+Linting is a key element of our wider approach to maintaining and improving the quality of our applications while also improving the efficiency of our software development workflows. Linters enable us to automate many of the checks that would otherwise be completed manually by PR reviewers, thereby shifting left to reduce the delay in the feedback loop and reducing the effort for the reviewer. Furthermore, linters, in combination with formatters, codify coding conventions and style guides, removing person preference and debate during peer reviews about minor style conventions.
```

Comment:
```suggestion
Linting is a key element of our wider approach to maintaining and improving the quality of our code while also improving the efficiency of our software development workflows. Linters enable us to automate many of the checks that would otherwise be completed manually by PR reviewers, thereby shifting left to reduce the delay in the feedback loop and reducing the effort for the reviewer. Furthermore, linters, in combination with formatters, codify coding conventions and style guides, removing person preference and debate during peer reviews about minor style conventions.
```